test/e2e_node/summary_test.go

```diff
@@ -124,7 +124,7 @@ var _ = SIGDescribe("Summary API [NodeConformance]", func() {
 				"WorkingSetBytes": bounded(10*e2evolume.Kb, memoryLimit),
 				"RSSBytes":        bounded(1*e2evolume.Kb, memoryLimit),
 				"PageFaults":      bounded(0, 1000000),
-				"MajorPageFaults": bounded(0, 10),
+				"MajorPageFaults": bounded(0, 1000),
```

Comment:
We should revisit this, since this is changing a conformance test, and results in different behavior between cg v1 and v2. I don't have it on top of my head, but I think this will always be `0` on cg v1 since the metric only counts page faults on the pod cgroup itself.